flowDeserializer.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/google/gopacket"
+)
+
+// Struct to be indexed in ElasticSearch
+type deserializedFlow struct {
+	Src          gopacket.Endpoint
+	Dest         gopacket.Endpoint
+	FlowHash     []byte
+	EndpointType gopacket.EndpointType
+}
+
```

Comment:
why is there a need for maintaining a different index for a flow

Reply:
Discussed at #6